modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,39 +1,51 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-importing-vmware-vm-to-ocp.adoc
+// * cnv/cnv_users_guide/cnv-create-vms.adoc
+
 [id="cnv-importing-vmware-vm_{context}"]
-= Importing a VMware virtual machine or template
+= Importing a VMware virtual machine or template with the virtual machine wizard
+
+You can import a VMware virtual machine or template. If you import a template, a virtual machine based on the template is created.
 
-You can use the {ProductName} web console to import a VMware virtual machine or template. If you import a template, a virtual machine based on the template is created.
+:FeatureName: Importing a VMware virtual machine or template
+include::modules/technology-preview.adoc[leveloffset=+1]
+:!FeatureName:
 
 .Prerequisites
 
 * The VMware virtual machine is powered off.
 * The VMware Virtual Disk Development Kit (VDDK) has been uploaded to the namespace in which you are importing the virtual machine.
 
-.Procedure
+include::cnv-uploading-vddk.adoc[leveloffset=+1]
```

Comment:
According to mod docs rules, modules should not include other modules like this. 

There might be another way to handle it, but since it seems like there are 3 procedures in this section, I'd have each of those be their own separate module, which is included in the assembly. And then in the assembly before these module inclusions, you can have some intro if needed. (Or you can put the prereqs in each module, if appropriate).

But since these are procedures, you do need the ".Procedure" heading in front of each of the set of steps.

Reply:
If I put the intro into the assembly, the heading will also have to be there. OK, if that's the only solution.